Modules/Server/Sources/PocketCastsServer/Public/Refresh/RefreshManager.swift

```diff
@@ -78,6 +82,40 @@ public class RefreshManager {
             }
         }
     }
+    #endif
+
+    #if os(watchOS)
```

Comment:
Not proud of this code duplication (triplication?). But I have no better idea :(